packages/web-lib/hooks/useBalances.tsx

```diff
@@ -77,38 +99,64 @@ function	useBalances(props?: Types.TUseBalancesReq): Types.TUseBalancesRes {
 				const	token = element.token as string;
 				const	balanceOf = results[rIndex++] as BigNumber;
 				const	decimals = results[rIndex++] as number;
-				const	symbol = results[rIndex++] as string;
-				const	price = results[rIndex++] as BigNumber;
+				let		symbol = results[rIndex++] as string;
```

Comment:
A lot of space between `let` and the var number 
```suggestion
				let symbol = results[rIndex++] as string;
```

Reply:
It would be nice to not have these casts, if there's a quick way to get rid of them I would